Flaky TestJdbcIntegrationSmokeTest

```
[ERROR] Tests run: 457, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 224.378 s <<< FAILURE! - in TestSuite
[ERROR] testExactPredicate(io.prestosql.plugin.jdbc.TestJdbcIntegrationSmokeTest)  Time elapsed: 0.025 s  <<< FAILURE!
java.lang.AssertionError: Execution of 'actual' query failed: SELECT * FROM orders WHERE orderkey = 10
	at org.testng.Assert.fail(Assert.java:83)
	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:147)
	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:103)
	at io.prestosql.testing.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:148)
	at io.prestosql.testing.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:143)
	at io.prestosql.testing.AbstractTestIntegrationSmokeTest.testExactPredicate(AbstractTestIntegrationSmokeTest.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: General error: "java.util.ConcurrentModificationException"; SQL statement:
SELECT TABLE_CATALOG TABLE_CAT, TABLE_SCHEMA TABLE_SCHEM, TABLE_NAME, TABLE_TYPE, REMARKS, TYPE_NAME TYPE_CAT, TYPE_NAME TYPE_SCHEM, TYPE_NAME, TYPE_NAME SELF_REFERENCING_COL_NAME, TYPE_NAME REF_GENERATION, SQL FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_CATALOG LIKE ?1 ESCAPE ?4 AND TABLE_SCHEMA LIKE ?2 ESCAPE ?4 AND TABLE_NAME LIKE ?3 ESCAPE ?4 AND TABLE_TYPE IN(?5, ?6) ORDER BY TABLE_TYPE, TABLE_SCHEM, TABLE_NAME [50000-199]
	at io.prestosql.testing.AbstractTestingPrestoClient.execute(AbstractTestingPrestoClient.java:114)
	at io.prestosql.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:379)
	at io.prestosql.testing.QueryAssertions.assertQuery(QueryAssertions.java:144)
	... 17 more
Caused by: io.prestosql.spi.PrestoException: General error: "java.util.ConcurrentModificationException"; SQL statement:
SELECT TABLE_CATALOG TABLE_CAT, TABLE_SCHEMA TABLE_SCHEM, TABLE_NAME, TABLE_TYPE, REMARKS, TYPE_NAME TYPE_CAT, TYPE_NAME TYPE_SCHEM, TYPE_NAME, TYPE_NAME SELF_REFERENCING_COL_NAME, TYPE_NAME REF_GENERATION, SQL FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_CATALOG LIKE ?1 ESCAPE ?4 AND TABLE_SCHEMA LIKE ?2 ESCAPE ?4 AND TABLE_NAME LIKE ?3 ESCAPE ?4 AND TABLE_TYPE IN(?5, ?6) ORDER BY TABLE_TYPE, TABLE_SCHEM, TABLE_NAME [50000-199]
	at io.prestosql.plugin.jdbc.BaseJdbcClient.getTableHandle(BaseJdbcClient.java:240)
	at io.prestosql.plugin.jdbc.jmx.StatisticsAwareJdbcClient.lambda$getTableHandle$3(StatisticsAwareJdbcClient.java:93)
	at io.prestosql.plugin.jdbc.jmx.JdbcApiStats.wrap(JdbcApiStats.java:35)
	at io.prestosql.plugin.jdbc.jmx.StatisticsAwareJdbcClient.getTableHandle(StatisticsAwareJdbcClient.java:93)
	at io.prestosql.plugin.jdbc.ForwardingJdbcClient.getTableHandle(ForwardingJdbcClient.java:75)
	at io.prestosql.plugin.jdbc.ForwardingJdbcClient.getTableHandle(ForwardingJdbcClient.java:75)
	at io.prestosql.plugin.jdbc.JdbcMetadata.getTableHandle(JdbcMetadata.java:81)
	at io.prestosql.plugin.jdbc.JdbcMetadata.getTableHandle(JdbcMetadata.java:52)
	at io.prestosql.metadata.MetadataManager.getTableHandle(MetadataManager.java:311)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:953)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:286)
	at io.prestosql.sql.tree.Table.accept(Table.java:53)
	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.analyzeFrom(StatementAnalyzer.java:2174)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:1160)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:286)
	at io.prestosql.sql.tree.QuerySpecification.accept(QuerySpecification.java:144)
	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:311)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:852)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:286)
	at io.prestosql.sql.tree.Query.accept(Query.java:107)
	at io.prestosql.sql.tree.AstVisitor.process(AstVisitor.java:27)
	at io.prestosql.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:301)
	at io.prestosql.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:278)
	at io.prestosql.sql.analyzer.Analyzer.analyze(Analyzer.java:83)
	at io.prestosql.sql.analyzer.Analyzer.analyze(Analyzer.java:75)
	at io.prestosql.execution.SqlQueryExecution.analyze(SqlQueryExecution.java:219)
	at io.prestosql.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:178)
	at io.prestosql.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:95)
	at io.prestosql.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:712)
	at io.prestosql.dispatcher.LocalDispatchQueryFactory.lambda$createDispatchQuery$0(LocalDispatchQueryFactory.java:119)
	at io.prestosql.$gen.Presto_327_49_gb882847__testversion____20201230_195916_3.call(Unknown Source)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
	... 3 more
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "java.util.ConcurrentModificationException"; SQL statement:
SELECT TABLE_CATALOG TABLE_CAT, TABLE_SCHEMA TABLE_SCHEM, TABLE_NAME, TABLE_TYPE, REMARKS, TYPE_NAME TYPE_CAT, TYPE_NAME TYPE_SCHEM, TYPE_NAME, TYPE_NAME SELF_REFERENCING_COL_NAME, TYPE_NAME REF_GENERATION, SQL FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_CATALOG LIKE ?1 ESCAPE ?4 AND TABLE_SCHEMA LIKE ?2 ESCAPE ?4 AND TABLE_NAME LIKE ?3 ESCAPE ?4 AND TABLE_TYPE IN(?5, ?6) ORDER BY TABLE_TYPE, TABLE_SCHEM, TABLE_NAME [50000-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.message.DbException.convert(DbException.java:347)
	at org.h2.command.Command.executeQuery(Command.java:219)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:115)
	at org.h2.jdbc.JdbcDatabaseMetaData.getTables(JdbcDatabaseMetaData.java:231)
	at io.prestosql.plugin.jdbc.BaseJdbcClient.getTables(BaseJdbcClient.java:687)
	at io.prestosql.plugin.jdbc.BaseJdbcClient.getTableHandle(BaseJdbcClient.java:221)
	... 40 more
Caused by: java.util.ConcurrentModificationException
	at java.util.HashMap$HashIterator.nextNode(HashMap.java:1445)
	at java.util.HashMap$ValueIterator.next(HashMap.java:1474)
	at org.h2.engine.Database.getAllTablesAndViews(Database.java:1769)
	at org.h2.table.MetaTable.getAllTables(MetaTable.java:700)
	at org.h2.table.MetaTable.generateRows(MetaTable.java:772)
	at org.h2.index.MetaIndex.find(MetaIndex.java:51)
	at org.h2.index.BaseIndex.find(BaseIndex.java:132)
	at org.h2.index.IndexCursor.find(IndexCursor.java:163)
	at org.h2.table.TableFilter.next(TableFilter.java:475)
	at org.h2.command.dml.Select$LazyResultQueryFlat.fetchNextRow(Select.java:1882)
	at org.h2.result.LazyResult.hasNext(LazyResult.java:101)
	at org.h2.result.LazyResult.next(LazyResult.java:60)
	at org.h2.command.dml.Select.queryFlat(Select.java:742)
	at org.h2.command.dml.Select.queryWithoutCache(Select.java:884)
	at org.h2.command.dml.Query.queryWithoutCacheLazyCheck(Query.java:151)
	at org.h2.command.dml.Query.query(Query.java:435)
	at org.h2.command.dml.Query.query(Query.java:397)
	at org.h2.command.CommandContainer.query(CommandContainer.java:145)
	at org.h2.command.Command.executeQuery(Command.java:202)
	... 44 more
```

In testExactPredicate: https://github.com/prestosql/presto/pull/2324/checks?check_run_id=368235944
In testDuplicatedRowCreateTable: https://github.com/prestosql/presto/pull/2323/checks?check_run_id=368439754

--- a/pom.xml
+++ b/pom.xml
@@ -692,7 +692,7 @@
             <dependency>
                 <groupId>com.h2database</groupId>
                 <artifactId>h2</artifactId>
-                <version>1.4.199</version>
+                <version>1.4.200</version>
             </dependency>
 
             <dependency>

